azure_pipelines/msal-release-ado-trigger.yml

```diff
@@ -6,4 +6,14 @@ trigger:
     include:
     - CHANGELOG.md
 
-pr: none
\ No newline at end of file
+pr: none
+
+jobs:
+- job:
+  displayName: "Trigger MSAL Release - OneBranch Pipeline"
+  steps:
+  - bash: |
+      az pipelines run --project $(ADOPROJECT) --id $(MSAL_ADO_RELEASE_PIPELINE_ID) --branch 'main' --organization $(ADOORGANIZATION)

```

Comment:
Nice! i didn't know you can do this from another pipeline. is azure cli always available?

Reply:
Yes, it is pre-installed on linux/windows agents, just like how xcode is pre-installed on macOS agents.